.github/workflows/ci.yml

```diff
@@ -14,9 +14,12 @@ jobs:
       with:
         go-version-file: go.mod
         check-latest: true
+
+    # Currently some tests rely on line endings so we only build
+    # without running.
     - run: go build ./...
 
-  darwin-build:
+  darwin-test:
```

Comment:
I realized these may have intentionally been only building and not testing before since there doesn't seem to be a reason for darwin to otherwise fail them. Let me know if I should revert these, but I thought it could be good to verify these work in CI

https://github.com/anuraaga/sqlc/commit/315cb5a6e4396112bbcc918821dffc1370191c7c

Reply:
FYI I have also filed https://github.com/pganalyze/pg_query_go/issues/103